bundle/Resources/views/ContentType/field_types.html.twig

```diff
@@ -183,18 +183,34 @@
         {{- form_widget(form.isMultiple) -}}
     </div>
 
-    <div class="ezselection-settings options{% if group_class is not empty %} {{ group_class }}{% endif %}">
+    <div id="{{form.options.vars.id}}-ezselection-root-options-list"
+         class="ezselection-settings options{% if group_class is not empty %} {{ group_class }}{% endif %}"
+         data-prototype="{{ ( form_widget(form.options.vars.prototype) ~ ' ' ~
```

Comment:
no! we can not do that, that's kind of ugly. But I think the whole topic deserves a bit more thinking/dev.